### PR TITLE
New Glue parameters

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/command-reference.md [hunk 1]
```diff
@@ -1535,22 +1540,31 @@ SYNOPSYS
...
 * **`--default-fs-override`** Provide an override for the default filesystem URI instead of a filesystem name (for example: `s3a://mybucket/`). This is referenced in the UI as **DefaultFS Override**.
 
+#### Glue Credential Parameters
+
+* **`--credentials-provider`** The [AWS catalog credentials provider factory class](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html#credentials-specify-provider). This is referenced in the UI as **AWS Catalog Credentials Provider**.  
+  * If this parameter is not provided, the default is [DefaultAWSCredentialsProviderChain](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/index.html?com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html)).
+  * If the `--access-key` and `--secret-key` parameters are provided, the credentials provider will automatically default to [StaticCredentialsProviderFactory](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/index.html?com/amazonaws/auth/AWSStaticCredentialsProvider.html).
+* **`--access-key`** The [AWS access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html). This is referenced in the UI as **Access Key**.
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:1557]: Line 1557 and subsequent parameters - I think it's overly complicated/formal to say that the parameter 'is referenced in the UI' and is quite obvious in most cases.

Would (Access Key in the UI) work, maybe after the parameter itself or after the link.

File: docs/command-reference.md [hunk 2]
```diff
@@ -1535,22 +1540,31 @@ SYNOPSYS
...
+* **`--secret-key`** The [AWS secret key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html). This is referenced in the UI as **Secret Key**.
+
+#### Glue Optional Parameters
+
+* **`--glue-catalog-id`** Provide the AWS Account ID that be used to access the Data Catalog. This is used if the Data Catalog is owned by a different account to that provided through the credentials provider and [cross-account access has been granted](https://docs.aws.amazon.com/glue/latest/dg/cross-account-access.html).
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:1562]: Line 1562 - Clean up 'to that provided': ..if the Data Catalog is owned by a different account to the one provided through....' or similar.

**Review by shonaisbister-disco — CHANGES_REQUESTED** [review]
Picky style-type comments from me!

File: docs/command-reference.md (hunk 2 above)
Comment (benradstone) [reply docs/command-reference.md:1562]: I was looking for a better way of writing this, thanks.

**Pull request merged**